src/ocaml_specific.ml

```diff
@@ -309,6 +312,16 @@ rule "ocaml C stubs: clib & (o|obj)* -> (a|lib) & (so|dll)"
         even if it is not the object file extension on your system."
   (C_tools.link_C_library "%(path)lib%(libname).clib" ("%(path)lib%(libname)"-.-ext_lib) "%(path)%(libname)");;
 
+rule "ocaml C stubs: (o|obj) -> clib"
+  ~prod:"%(path:<**/>)lib%(libname:<*> and not <*.*>).clib"
+  ~dep:("%(path)%(libname)"-.-ext_obj)
+  ~doc:"The preferred way to build a C library libfoo.a is to create \
```

Comment:
Is it really `preferred`? I don't see a reason to create a `clib` if you just have one file, much like `mllib`.